sDownloads.ps1

```diff
@@ -0,0 +1,27 @@
+# Check directory sDownloads.ps1 runs from for sd-config file
+$ConfigPath = $PSScriptRoot + "\sd-config"
```

Comment:
This is good, but there may be a better way to do it. I attempted to run this line-by-line and so the $PSScriptRoot Env Var wasn't available to me.